test/Microsoft.Extensions.DependencyModel.Tests/DependencyContextBuilderTests.cs

```diff
@@ -176,6 +178,9 @@ public void FillsRuntimeLibraryProperties()
             lib.Dependencies.Should().OnlyContain(l => l.Name == "System.Collections" && l.Version == "3.3.3");
             lib.ResourceAssemblies.Should().OnlyContain(l => l.Path == "en-US/Pack.Age.resources.dll" && l.Locale == "en-US");
 
+            // When ProjectModel supports path in the lock file library, this should assert "path/TO/package".
```

Comment:
That was completely unintentional. When I moved the tests over I guess I missed some. These should go back to CLI, the ones for projectmodel. But I will do that eventually.